teamsupport/utils.py

```diff
@@ -0,0 +1,22 @@
+from lxml import etree
+
+
+def to_xml(root, data=None, cdata=None):
+    xml = etree.Element(root)
+
+    if data:
+        for k, v in data.iteritems():
+            child = etree.SubElement(xml, k)
+            child.text = v
+
+    if cdata:
```

Comment:
Urgh. Whether a string is literal in XML, or in a CDATA block, should be entirely transparent to the consumer. If their API requires something to be in a CDATA block, they're idiots.